### ISSUE TITLE
function_clause in EDoc extraction

### ISSUE THREAD

**Issue by erszcz** [erszcz/docsh#2]
```
$ erl
Erlang/OTP 19 [erts-8.0] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false]

Eshell V8.0  (abort with ^G)
1> h(fun proplists:get_value/3).
docsh error: function_clause
[{docsh_edoc_xmerl,unwrap_inline,
                   [{fmt,[["  - ",["expand([{foo, [bar, baz]}],","\n"]],
                          ["    ",["[fie, foo, fum])","\n"]],
                          ["  - ",
                           ["expand([{{foo, true}, [bar, baz]}],","\n"]],
                          ["    ",["[fie, foo, fum])","\n"]],
                          ["  - ",
                           ["expand([{{foo, false}, [bar, baz]}],","\n"]],
                          ["    ",["[fie, {foo, false}, fum])","\n"]]]}],
                   [{file,"/home/erszcz/work/erszcz/docsh/_build/default/lib/docsh/src/docsh_edoc_xmerl.erl"},
                    {line,102}]},
 {docsh_edoc_xmerl,'-cleanup_lines/1-lc$^1/1-1-',1,
                   [{file,"/home/erszcz/work/erszcz/docsh/_build/default/lib/docsh/src/docsh_edoc_xmerl.erl"},
                    {line,117}]},
 {docsh_edoc_xmerl,'-cleanup_lines/1-lc$^1/1-1-',1,
                   [{file,"/home/erszcz/work/erszcz/docsh/_build/default/lib/docsh/src/docsh_edoc_xmerl.erl"},
                    {line,117}]},
 {docsh_edoc_xmerl,cleanup_lines,1,
                   [{file,"/home/erszcz/work/erszcz/docsh/_build/default/lib/docsh/src/docsh_edoc_xmerl.erl"},
                    {line,117}]},
 {docsh_edoc_xmerl,'#element#',5,
                   [{file,"/home/erszcz/work/erszcz/docsh/_build/default/lib/docsh/src/docsh_edoc_xmerl.erl"},
                    {line,90}]},
 {xmerl,tagdef,5,[{file,"xmerl.erl"},{line,261}]},
 {xmerl,export_content,2,[{file,"xmerl.erl"},{line,199}]},
 {xmerl,export_content,2,[{file,"xmerl.erl"},{line,191}]}]

-spec get_value(Key, List, Default) -> term() when Key :: term(),
                                                   List :: [term()],
                                                   Default :: term().

ok
2> 
BREAK: (a)bort (c)ontinue (p)roc info (i)nfo (l)oaded
       (v)ersion (k)ill (D)b-tables (d)istribution
^C00:24:09 erszcz @ x2 : ~/work/erszcz/docsh (master %)
$ git l -1
f7f3fd1 Add beamfile:decompile/1
```

**Comment by erszcz** [comment]
No longer a problem as of f58b3a9 - closing.

**Comment by erszcz** [comment]
Rechecked, not fixed as of ecf3582. Reopening.

**Comment by erszcz** [comment]
Duplicate with https://github.com/erszcz/docsh/issues/17. Closing.
